docs/src/model_setup/buoyancy_and_equation_of_state.md

```diff
@@ -43,10 +47,32 @@ NonhydrostaticModel{CPU, Float64}(time = 0 seconds, iteration = 0)
 └── coriolis: Nothing
 ```
 
+
+In order to create a `HydrostaticFreeSurfaceModel` without any buoyancy treatment we need to pass
+explicitly pass the `buoyancy=nothing` flag. Note that by default `HydrostaticFreeSurfaceModel`
+advects temperature `T` and salinity `S` (which aren't necessary without a buoyancy treatment), so
+it is often recommended to explicitly specify the tracers in this case as well:
+
+```jldoctest buoyancy
+julia> model = HydrostaticFreeSurfaceModel(grid=grid, buoyancy=nothing, tracers=())
+┌ Warning: HydrostaticFreeSurfaceModel is experimental. Use with caution!
+└ @ Oceananigans.Models.HydrostaticFreeSurfaceModels ~/builds/tartarus-3/clima/oceananigans/src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl:106
```

Comment:
This can't be part of a doctest. It won't work. It's machine specific. Also, sometimes it'll be `~/builds/tartarus-3/clima/...` or `~/builds/tartarus-2/clima/...` etc.

The proper solution involves Doctest filtering, see
https://juliadocs.github.io/Documenter.jl/stable/man/doctests/#Filtering-Doctests

Reply:
Actually, I just noticed that the docs build is failing exactly because of this.

Reply:
Done. Thanks again for opening #2080